login/handler_test.go

```diff
@@ -455,11 +455,30 @@ func TestHandler_getToken_Valid(t *testing.T) {
 	r := &http.Request{
 		Header: http.Header{"Cookie": {h.config.CookieName + "=" + token + ";"}},
 	}
-	userInfo, valid := h.GetToken(r)
+	userInfo, _, valid := h.GetToken(r)
 	True(t, valid)
 	Equal(t, input, userInfo)
 }
 
+func TestHandler_getToken_Key_ID(t *testing.T) {
```

Comment:
The Name of the Test does not describe the tested feature. It should state that the `kid` value is stored in the jwt header if a token is generated.